pennylane/ops/qubit/matrix_ops.py

```diff
@@ -513,22 +515,27 @@ class BlockEncode(Operation):
     """Gradient computation method."""
 
     def __init__(self, A, wires, id=None):
+        wires = Wires(wires)
         shape_a = qml.math.shape(A)
         if shape_a == () or all(x == 1 for x in shape_a):
             A = qml.math.reshape(A, [1, 1])
-
-        wires = Wires(wires)
-        if pnp.sum(shape_a) <= 2:
-            normalization = A if A > 1 else 1
+            normalization = qml.math.abs(A)
```

Comment:
`abs` and other `qml.math` methods can be imported similar to those imported methods in line 24 for consistency.

Reply:
+1